.gitignore

```diff
@@ -8,6 +8,7 @@ npm-debug.log
 *.sublime-*
 coverage
 .DS_Store
+.mason
```

Comment:
Huh. I see we have a similar line in `.npmignore`. Where did this come from? What causes a `.mason` directory to be created?